src/axom/primal/operators/is_convex.hpp

```diff
@@ -54,7 +56,7 @@ bool is_convex(const Polygon<T, 2>& poly, double EPS = 1e-8)
     }
 
     // Ensure other point to check against isn't adjacent
-    if(res1 == orientation(poly[(i < n / 2) ? n : 0], seg, EPS))
+    if(res1 == orientation(poly[(i <= n / 2) ? n : 0], seg, EPS))
```

Comment:
Is there an associated test that would fail with the previous formulation but not the new one?

Reply:
There definitely was at one point, it must have gotten removed when I made the new branch separate from the ray casting winding number methods. I'll re-add it. Thanks for pointing that out!